File: examples/rss/add/node/addANewFeed.js
const { RssService } = require("m3o/rss");

// Add a new RSS feed with a name, url, and category
async function addAnewFeed() {
  let rssService = new RssService(process.env.MICRO_API_TOKEN);
  let rsp = await rssService.add({
    category: "news",
    name: "bbc",
    url: "http://feeds.bbci.co.uk/news/rss.xml",
  });
  console.log(rsp);
}

addAnewFeed();

File: examples/go/rss/add/addANewFeed.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/rss"
)

// Add a new RSS feed with a name, url, and category
func AddAnewFeed() {
	rssService := rss.NewRssService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := rssService.Add(&rss.AddRequest{
		Category: "news",
		Name:     "bbc",
		Url:      "http://feeds.bbci.co.uk/news/rss.xml",
	})
	fmt.Println(rsp, err)
}
